test/unit-tests/components/views/elements/AppTile-test.tsx

```diff
@@ -106,27 +108,42 @@ describe("AppTile", () => {
             if (roomId === "r2") return [app2];
             return [];
         });
+    });
+
+    afterAll(async () => {
+        jest.restoreAllMocks();
+    });
+
+    beforeEach(async () => {
+        // Do not carry across settings from previous tests
+        SettingsStore.reset();
+        sdkContext = new SdkContextClass();
+        // @ts-ignore
+        await WidgetMessagingStore.instance.onReady();
 
         // Wake up various stores we rely on
         WidgetLayoutStore.instance.useUnitTestClient(cli);
         // @ts-ignore
         await WidgetLayoutStore.instance.onReady();
+
         RightPanelStore.instance.useUnitTestClient(cli);
         // @ts-ignore
         await RightPanelStore.instance.onReady();
     });
 
-    beforeEach(async () => {
-        sdkContext = new SdkContextClass();
+    afterEach(async () => {
         jest.spyOn(SettingsStore, "getValue").mockRestore();
         // @ts-ignore
+        await WidgetLayoutStore.instance.onNotReady();
+        // @ts-ignore
+        await RightPanelStore.instance.onNotReady();
+        // @ts-ignore
         await WidgetMessagingStore.instance.onReady();
```

Comment:
Since this is now duplicated in `beforeEach`:
```suggestion
        await WidgetMessagingStore.instance.onNotReady();
```